roles/setup-slack/tasks/invite_users.yml

```diff
@@ -1,14 +1,14 @@
 ---
 
 - name: Get channels info
-  include: get_channel_info.yml
+  include_tasks: get_channel_info.yml
```

Comment:
This should be `import_tasks` as it's not a dynamic include

Reply:
Good catch, thanks!